logic/src/commonTest/kotlin/com/wire/kalium/logic/data/register/RegisterAccountRepositoryTest.kt

```diff
@@ -118,35 +118,60 @@ class RegisterAccountRepositoryTest {
         val ssoId = with(TEST_USER.ssoID) {
             this?.let { SsoId(scimExternalId = it.scimExternalId, subject = it.subject, tenant = it.tenant) }
         }
+        val cookieLabel = "COOKIE_LABEL"
         val authTokens = with(SESSION) {
             AuthTokens(
                 userId = UserId(userId.value, userId.domain),
                 accessToken = accessToken,
                 refreshToken = refreshToken,
-                tokenType = tokenType
+                tokenType = tokenType,
+                cookieLabel = cookieLabel
             )
         }
         val expected = Pair(ssoId, authTokens)
 
         given(registerApi).coroutine {
             register(
-                RegisterApi.RegisterParam.PersonalAccount(email, code, name, password)
+                RegisterApi.RegisterParam.PersonalAccount(
+                    email = email,
+                    emailCode = code,
+                    name = name,
+                    password = password,
+                    cookieLabel = cookieLabel
+                )
```

Comment:
I see here so much potential to adapt to `Arrangement` pattern and save us some precious extra lines  😅

Reply:
i think so too by i will leave it for another PR